content/en/tracing/trace_collection/dd_libraries/dotnet-core.md

```diff
@@ -66,11 +66,75 @@ For a full list of supported libraries and processor architectures, see [Compati
 
 ### Installation
 
-1. [Install the tracer.](#install-the-tracer)
-2. [Enable the tracer for your service.](#enable-the-tracer-for-your-service)
-3. [Configure the Datadog Agent for APM.](#configure-the-datadog-agent-for-apm)
+1. [Configure the Datadog Agent for APM.](#configure-the-datadog-agent-for-apm)
+2. [Install the tracer.](#install-the-tracer)
+3. [Enable the tracer for your service.](#enable-the-tracer-for-your-service)
 4. [View your live data.](#view-your-live-data)
 
+### Configure the Datadog Agent for APM
```

Comment:
Should this part be moved to another page. I assume we should not repeat it for each tracer

Reply:
Yes, we intentionally repeat it for each tracer. Some users will only use one set of instructions, we want them to get a full picture here and minimize the clicking back and forth they need to do. In this case, we duplicate content to reduce the labor of navigating docs. There are definitely grey areas to this strategy, but I think this one is solid.